### PR TITLE
[WIP] Updating descriptions for community bots

### PR DESCRIPTION
**Pull request by Piterden**
Guys, please write a descriptions for your bots to this PR

@TiagoDanin @backmeupplz

There are only your bots left

### CODE REVIEW

File: docs/README.md [hunk 1]
```diff
@@ -57,50 +57,52 @@ For additional bot examples see [`examples`](https://github.com/telegraf/telegra
...
+| [DefendTheCastle](https://github.com/TiagoDanin/Defend-The-Castle) |   |
+| [EveMoviesBot](https://github.com/dmbaranov/evemovies-bot) |   |
+| [GNU/LinuxIndonesiaBot](https://github.com/bgli/bglibot-js) |   |
+| [GoogleItBot](https://github.com/Edgar-P-yan/google-it-telegram-bot) | Instant inline search |
+| [GroupsAdminBot](https://github.com/Azhant/AdminBot) |   |
+| [KitchenTimerBot](https://github.com/DZamataev/kitchen-timer-bot) |   |
+| [LyricsGramBot](https://github.com/lioialessandro/LyricsGramBot) | Song Lyrics |
+| [MangadexBot](https://github.com/ejnshtein/mangadex_bot) |   |
+| [Memcoin](https://github.com/backmeupplz/memcoin) |   |
+| [MetalArchivesBot](https://github.com/amiralies/metalarchives-telegram-bot) |   |
```
Comment (amiralies) [inline docs/README.md]: sth like `Unofficial metal-archives.com telegram bot`

Comment (Piterden) [reply docs/README.md]: Could you push it pls?

Comment (Piterden) [reply docs/README.md]: To this [branch](https://github.com/Piterden/telegraf/tree/patch-3)

Comment (amiralies) [reply docs/README.md]: access?

Comment (Piterden) [reply docs/README.md]: Make a PR, I will merge

Comment (Piterden) [reply docs/README.md]: Make a PR

**Comment by Piterden** [comment]
Come on guys, lets cheat a bit! )))

File: docs/README.md (hunk 1 above)
Comment (Piterden) [reply docs/README.md]: Nice thnx

**Comment by lioialessandro** [comment]
Hi!
The description for my bot (LyricsGramBot) is already present

**Comment by Piterden** [comment]
Got it, sorry for ping... @lioialessandro

**Comment by Edgar-P-yan** [comment]
@Piterden Hi!
My [GoogleItBot](https://github.com/Edgar-P-yan/google-it-telegram-bot) already has a description

**Comment by Piterden** [comment]
@TiagoDanin @backmeupplz

There are only your bots left

**Comment by Piterden** [comment]
@dotcypress I think we could merge it. There are only a few bots without description left in the list...

**Pull request merged**